### DISCUSSION TITLE
Waterfox compatibility

### DISCUSSION THREAD

**Discussion by cOborski** [filips123/PWAsForFirefox#456] [Q&A] [answered]
**Has anyone ever successfully got PWAsForFirefox working with Waterfox?**

I happen to use Waterfox, and for the most part I forget I'm not using standard Firefox, except occasionally, I run into a browser add-on I can't use. Normally, I just move on, but I'd like to get PWAsForFirefox working with Waterfox.

I haven't done any deep investigating, other than a standard web search (which didn't turn anything up). I thought I'd start by asking here. Once I figure out why it doesn't work, I'll create an issue.

### Waterfox
https://www.waterfox.net/
https://www.waterfox.net/docs/about-waterfox/
### Waterfox Portable (unaffiliated with official Waterfox)
https://portapps.io/app/waterfox-portable/
### Scoop
https://scoop.sh/#/apps?q=waterfox

**Comment by filips123** [comment, marked as answer]
Which issue exactly do you have? Does the addon not detect the native connector even if it's installed?

Also, which Waterfox version do you have? Their changelog says that native messaging should be supported automatically since G5.0 Beta 3.

**Reply by cOborski** [reply]
I jumped the gun on this--everything worked fine when I can back to it the next day.
